oc-chef-pedant/lib/pedant/platform.rb

```diff
@@ -195,18 +195,18 @@ def create_client(name, org = self.test_org)
       payload = { "name" => clientname }
 
       if server_api_version == 0
-        r = post(api_url('/clients'), org.validator, :payload => payload)
+        r = post(api_url('/clients', org), org.validator, :payload => payload)
```

Comment:
Interesting, what API requests were these being made to without passing an org? You commit message mentions the default_org.  Do you mean the default org as configured for the chef-server under test?  Does that imply these tests previously polluted a user's organization if run on a live cluster?